src/Spotify.js

```diff
@@ -0,0 +1,34 @@
+import { URL } from 'url';
+
+const podcastTypes = [
+  'album',
+  'artist',
+  'episode',
+  'show',
+  'track',
+  'playlist',
+];
+
+export const shouldTransform = url => {
+  const { host, pathname } = new URL(url);
+  const pathnameFields = pathname.split('/');
+  const type =
+      (pathnameFields && pathnameFields[1] && pathnameFields[1].toLowerCase()) ||
+      null;
+  return host === 'open.spotify.com' && podcastTypes.includes(type);
+};
+
+export const getSpotifyIFrameSrc = urlString => {
+  const { pathname } = new URL(urlString);
+  const type = pathname.split('/')[1].toLowerCase();
+
+  return (
+      podcastTypes.includes(type) && urlString.replace(type, `embed/${type}`)
```

Comment:
Here you have to insert `embed/` before the `type`, except for when `type` is `episode` or `show`, then you'll have to insert `embed-podcast/`.

```js
const podcastTypes = ['episode', 'show'];
if(podcastTypes.includes(type)) {
  return urlString.replace(type, `embed-podcast/${type}`);
}

return urlString.replace(type, `embed/${type}`);
```

Reply:
Hey I checked this, is there any difference between the both. UI was identical and there were no difference between `embed` & `embed-podcast`

Reply:
If you use take the `embed` url and past it into the browser, you'll see it's redirected to `embed-post`.

Reply:
I see. Will do this might have missed this